content/en/docs/guides/server-graceful-stop.md

```diff
@@ -0,0 +1,86 @@
+---
+title: GracefulStop
```

Comment:
We would want the title to be "Graceful Stop".

Reply:
I would suggest "Gracefully stopping a server"

Reply:
Change to "Gracefully stopping a server"

Reply:
`Graceful Shutdown`?

If you look at the website preview, this becomes the second guide that word wraps.  I'd rather avoid that if at all possible.

Reply:
Changed to Graceful Shutdown